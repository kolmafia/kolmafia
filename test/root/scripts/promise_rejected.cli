js new Promise((resolve, reject) => reject("bar"))
