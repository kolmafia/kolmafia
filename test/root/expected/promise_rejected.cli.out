JavaScript exception: bar
Returned: null